src/ontology/cl-edit.owl

```diff
@@ -17036,7 +17036,7 @@ SubClassOf(obo:CL_0002324 obo:CL_0000185)
 
 # Class: obo:CL_0002325 (lactocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:19022771") Annotation(oboInOwl:hasDbXref "PMID:28717367") Annotation(oboInOwl:hasDbXref "PMID:35091553") Annotation(oboInOwl:hasDbXref "PMID:35377806") Annotation(oboInOwl:hasDbXref "doi:10.1177/156482659601700404") obo:IAO_0000115 obo:CL_0002325 "A milk-producing glandular epithelial cell that is part of a mammary gland alveolus and differentiates from a luminal adaptive secretory precursor cell during lactogenesis I. Following secretory activation during lactogenesis II, a lactocyte is involved in the biosynthesis and/or concentration of milk oligosaccharides, lactose, micronutrients, fat, hormones, immunoglobulins, and cytokines, as well as their transport into the lumen of the lactating mammary gland.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:19022771") Annotation(oboInOwl:hasDbXref "PMID:28717367") Annotation(oboInOwl:hasDbXref "PMID:35091553") Annotation(oboInOwl:hasDbXref "PMID:35377806") Annotation(oboInOwl:hasDbXref "doi:10.1177/156482659601700404") obo:IAO_0000115 obo:CL_0002325 "A milk-producing glandular epithelial cell that is part of a mammary gland alveolus and differentiates from a luminal adaptive secretory precursor cell during secretory differentiation (also termed lactogenesis I). Following secretory activation (also termed lactogenesis II), a lactocyte is involved in the synthesis and/or transport of milk constituents including proteins, oligosaccharides, lactose, micronutrients, fat, hormones, immunoglobulins , and cytokines into the lumen of the lactating mammary gland.")
```

Comment:
- "... immunoglobulins , and... " --> "immunoglobulins, and" or "immunoglobulins and"